### ISSUE TITLE
526 Form Validation

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#54285]
## High Level User Story/ies
As a veteran or a representative of a veteran, I want to be able to file benefit claims accurately and successfully. 
### Current:
Most forms on the core submission flow allow user to enter values that are not accepted by downstream systems that establish claims or consume information collected via form 526 and cause the submission process to fail.
Attached is the current list of errors identified and number of occurrences for each error.
 
### Future:
Implement preventative measures that both educate the user on correct values to enter and do not allow incorrect values to be submitted. 

## Hypothesis or Bet
If we make this change, we expect benefit claims filed by a veteran or representative via va.gov to flow through the system and get established as a claim on VBMS.

## OKR
_Which Objective / Key Result does this epic push forward?_

### What must be true in order for you to consider this epic complete? 
- Where unclear, user gets actionable information on each form on the acceptable value for a field
- The 526 form does not allow a claim to be submitted with values that will not be accepted by downstream systems for establishing the claim
- Accessibility criteria - 

*Take into consideration Accessibility/QA needs as well as Product, Technical, and Design requirements.*

## How to configure this issue
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `Console-Services`, `tools-fe`,`content-ia`, `service-design`, `vsa-*`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)

**Comment by NB28VT** [comment]
Let me know if there is a better place to note this, but I noticed when going through the form one of the validation messages for email lists [regular expressions](https://medium.com/factory-mind/regex-tutorial-a-simple-cheatsheet-by-examples-649dc1c3f285) as acceptable values, which is meaningless to most non-programmers (and even many programmers). So these should be replaced with more useful examples such as "example@email.com" etc.

![Screenshot 2023-05-26 at 12.04.30 PM.png](https://images.zenhubusercontent.com/647627f92b5e617343beedd1/9f86e8ab-41dc-40b5-9017-277b3c20891b)

**Comment by RakshindaAslam** [comment]
Hi @NB28VT - It's fine to note it here but the outcome of this would be a ticket on the Zenhub board to fix each of these issues associated with this epic. 
@austinjprice - do you all have a process for creating new tickets?

**Comment by austinjprice** [comment]
@austinjprice to digest what has been completed and propose next steps

**Comment by austinjprice** [comment]
research timebox - 2 half points

**Comment by austinjprice** [comment]
research timebox - 2 half points

**Comment by austinjprice** [comment]
@nnicholas7 updated list of error and warnings from last six months of failed submissions located here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/black_hole_form_validation_research.md

**Comment by lisacapaccioli** [comment]
Closing this as it's no longer needed.
